falkordb-cluster/cluster-entrypoint.sh

```
@@ -74,6 +74,23 @@ if [[ $OMNISTRATE_ENVIRONMENT_TYPE != "PROD" ]];then
   DEBUG=1
 fi
 
+
+
+update_ips_in_nodes_conf(){
+  if [[ -f "$DATA_DIR/nodes.conf" && -s "$DATA_DIR/nodes.conf" ]];then
+    res=(cat $DATA_DIR/nodes.conf | grep myself | awk '{print $2}' | cut -d',' -f1)
```

Shellcheck (0.10.0)

[error] Couldn't parse this array assignment. Fix to allow more checks.

(SC1073)